tools/stats/upload_external_contrib_stats.py

```diff
@@ -0,0 +1,114 @@
+import os
+import urllib.parse
+from typing import (
+    Any,
+    Callable,
+    Dict,
+    Iterable,
+    List,
+    Optional,
+    Pattern,
+    Tuple,
+    Union,
+    cast,
+    NamedTuple
+)
+import json
+from urllib.error import HTTPError
+from urllib.request import Request, urlopen
+import datetime
+import time
+from upload_stats_lib import upload_to_s3
+import argparse
+
+FILTER_OUT_USERS = set(["raghuramank100", "facebook-github-bot"])
```

Comment:
I'd update to 
`FILTER_OUT_USERS = set(["pytorchmergebot", "facebook-github-bot", "pytorch-bot[bot]"])`

(raghuramank100 is no longer relevant, also remove `-author:pytorchmergebot` from line 69 and use the one from `FILTER_OUT_USERS` for maintainabilty)